index.html

```diff
@@ -57,6 +57,174 @@ <h2>I`am a Software Developer</h2>
+              <ul><li class="list2">Html</li>
+                <li class="list2">Bootstrap</li>
+                <li class="list2">Ruby </li>
+              </ul>
+            </div>
+            <div class="work2-button"><button type="button">See Project</button></div>
+        </div>
+        <div class="work-2">
+          <div class="work2-content">
+            <h1>Profesional Art Printing Data</h1>
+            <h2>Profesional Art Printing Data</h2>
+            <p>First A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry`s standard</p>
+            <ul><li class="list2">Html</li>
+              <li class="list2">Bootstrap</li>
+              <li class="list2">Ruby </li>
+            </ul>
+          </div>
+          <div class="work2-button"><button type="button">See Project</button></div>
+      </div>
+      <div class="work-2">
+        <div class="work2-content">
+          <h1>Website Protfolio</h1>
+          <h2>Website Protfolio</h2>
+          <p>Sixth A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry`s standard</p>
+          <ul>
+            <li class="list2">Html</li>
+            <li class="list2">Bootstrap</li>
+            <li class="list2">Ruby </li>
+          </ul>
+        </div>
+        <div class="work2-button"><button type="button">See Project</button></div>
+      </div>
+        </div>
+          <div class="popUp-Container"></div>
+        </section>
+        <section class="about-me-section" id="about">
+          <div class="about-me-1">
+            <div class="about-me1-content">
+              <h1>About me</h1>
+              <p>
+                Hello I`m a software developer!  I can help you build a product ,
+                feature or website Look through some of my work and experience! If
+                you like what you see and have a project you need coded, don`t
+                hestiate to contact me.
+              </p>
+              <div class="about-me-btn"><input type="button" class="about-me-button" value="Get my Resume"></div>
+            </div>
+            <div class="about-me-background" alt="Backgroud Image"></div>
+          </div>
+          <div class="about-me1-separation">
+            <div class="line2"></div>
+          </div>
+          <ul class="about-me-2">
+            <li class="grid1">
+              <div class="topic1">
+                <img src="./svg/language.svg" alt="Language" />
+                <p>Language</p>
+              </div>
+              <ul class="discussion">
+                <li><a href="https://en.wikipedia.org/wiki/JavaScript">Javascript</a></li>
+                <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+                <li><a href="https://en.wikipedia.org/wiki/HTML">HTML</a></li>
+                <li><a href="https://en.wikipedia.org/wiki/CSS">CSS</a></li>
+              </ul>
+            </li>
```

Comment:
[Optional] As the previous reviewer requested to align your skills section properly you did great. But there is some improvement you can consider:

- [ ] First, add space between the list items in the list to improve readability and visual separation. Consider adding top and bottom padding to each list item.
![image](https://github.com/sulaimanahmadi/Portfolio/assets/55840999/861b5020-fb8b-4568-bfdf-2715fcd52a9d)

- [ ] Then apply a gap between all list items by adding `gap: 8px;` to the .discussion class. This will create consistent spacing between the skills.
![image](https://github.com/sulaimanahmadi/Portfolio/assets/55840999/0c2ee230-cde2-45dd-82e2-400c5833ec81)

Note: Please follow the same instructions for **Language**, **Frameworks** and **Skills** section